pom.xml

```diff
@@ -250,8 +249,8 @@
     <dependency>
       <groupId>org.kohsuke</groupId>
       <artifactId>access-modifier-annotation</artifactId>
-      <version>${access-modifier-checker.version}</version>
       <scope>provided</scope>
+      <optional>true</optional>
```

Comment:
I'm not quite sure why the other annotation dependencies have both `<scope>provided</scope>` _and_ `<optional>true</optional>`, but I figured I might as well make this one consistent with them.